src/vlogger/vlogger.h

```diff
@@ -80,15 +80,27 @@
 #define __log_warn(log_fmt, log_args...)         do { VLOG_PRINTF(VLOG_WARNING, log_fmt, ##log_args); } while (0)
 #define __log_info(log_fmt, log_args...)         do { VLOG_PRINTF(VLOG_INFO, log_fmt, ##log_args); } while (0)
 
-#if defined(VMA_OPTIMIZE_LOG)
+#if defined(VMA_OPTIMIZE_LOG) && (VMA_OPTIMIZE_LOG <= 4)
 #define __log_details(log_fmt, log_args...)      ((void)0)
-#define __log_dbg(log_fmt, log_args...)          ((void)0)
-#define __log_fine(log_fmt, log_args...)         ((void)0)
-#define __log_finer(log_fmt, log_args...)        ((void)0)
 #else
 #define __log_details(log_fmt, log_args...)      do { if (g_vlogger_level >= VLOG_DETAILS) 	VLOG_PRINTF(VLOG_DETAILS, log_fmt, ##log_args); } while (0)
+#endif
+
+#if defined(VMA_OPTIMIZE_LOG) && (VMA_OPTIMIZE_LOG <= 5)
```

Comment:
Why asking twice on VMA_OPTIMIZE_LOG
```
defined(VMA_OPTIMIZE_LOG) && (VMA_OPTIMIZE_LOG <= 5)
```
instead of asking about it once?

Maybe VMA_OPTIMIZE_LOG should be initialized to a high value?